config/environments/docker.rb

```diff
@@ -0,0 +1,73 @@
+require 'active_support/core_ext/integer/time'
```

Comment:
As mention later in another comment, we should probably find a way to run every environment in the docker container.